### PR TITLE
Setup React

### PR DESCRIPTION
**Pull request by Lev-Stambler**
Setup the general React outline as per @cwlarson-google request

### CODE REVIEW

File: biogrid/libs/components/src/lib/components.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './components.css';
+
+/* eslint-disable-next-line */
```
Comment (cwlarson-google) [inline biogrid/libs/components/src/lib/components.tsx]: What rule are you disabling here?

File: biogrid/libs/components/src/lib/components.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (cwlarson-google) [inline biogrid/libs/components/src/lib/components.tsx]: What's the use case for components.tsx? Is this just an example of how to write a component?

**Review by cwlarson-google — APPROVED** [review]

File: biogrid/apps/api/src/main.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 import { Message } from '@biogrid/api-interfaces';
 
 const app = express();
 
 const greeting: Message = { message: 'Welcome to api!' };
 
-app.get('/api', (req, res) => {
-  res.send(greeting);
+app.get('/api', (req: unknown, res: unknown) => {
+  // res.send(greeting);
```
Comment (vitashubin) [inline biogrid/apps/api/src/main.ts]: uncomment or remove?

File: biogrid/apps/biogrid-mvp/src/app/pages/infopage/infopage.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import InfoPage from './infopage';
+
+describe('Info page', () => {
+  afterEach(() => {
+    delete global['fetch'];
+    cleanup();
+  });
+
+  it('should render successfully', async () => {
+    global['fetch'] = jest.fn().mockResolvedValueOnce({
```
Comment (vitashubin) [inline biogrid/apps/biogrid-mvp/src/app/pages/infopage/infopage.spec.tsx:13]: nit, have 'my message' as a const variable that is reused in this mockResolvedValueOnce and in the await in line 19

File: biogrid/libs/components/src/lib/components.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Components from './components';
+
+describe(' Components', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<Components />);
+    // expect(baseElement).toBeTruthy();
```
Comment (vitashubin) [inline biogrid/libs/components/src/lib/components.spec.tsx]: why is this commented out? If it is removed, this isn't really testing anything right?

**Review by vitashubin — CHANGES_REQUESTED** [review]

File: biogrid/libs/components/src/lib/components.spec.tsx (hunk 5 above)
Comment (Lev-Stambler) [reply biogrid/libs/components/src/lib/components.spec.tsx]: Oh yes, I was having linter issues, I will put that back in

File: biogrid/libs/components/src/lib/components.tsx (hunk 1 above)
Comment (Lev-Stambler) [reply biogrid/libs/components/src/lib/components.tsx]: Auto added, will remove

File: biogrid/libs/components/src/lib/components.tsx (hunk 2 above)
Comment (Lev-Stambler) [reply biogrid/libs/components/src/lib/components.tsx]: Yep, it was auto added but I can make a different component example which can further better outline the components

File: biogrid/apps/api/src/main.ts (hunk 3 above)
Comment (Lev-Stambler) [reply biogrid/apps/api/src/main.ts]: My bad!

File: biogrid/libs/components/src/lib/components.tsx (hunk 1 above)
Comment (Lev-Stambler) [reply biogrid/libs/components/src/lib/components.tsx]: Oh the linter flags empty interfaces, I added a property

File: biogrid/libs/components/src/lib/components.tsx (hunk 2 above)
Comment (Lev-Stambler) [reply biogrid/libs/components/src/lib/components.tsx]: I changed it to be an example header component

File: biogrid/apps/api/src/main.ts (hunk 3 above)
Comment (Lev-Stambler) [reply biogrid/apps/api/src/main.ts]: Uncommented

File: biogrid/libs/components/src/lib/components.spec.tsx (hunk 5 above)
Comment (Lev-Stambler) [reply biogrid/libs/components/src/lib/components.spec.tsx]: Added back, it was because of a linter issue. Fixed now

**Review by vitashubin — APPROVED** [review]
Thanks for making the changes, looks great!

**Pull request merged**